src/ftxui/component/component.cpp

```diff
@@ -166,7 +166,7 @@ bool ComponentBase::Focused() const {
 /// @brief Make the |child| to be the "active" one.
 /// @param child the child to become active.
 /// @ingroup component
-void ComponentBase::SetActiveChild(ComponentBase* /*child*/) {}
+void ComponentBase::SetActiveChild([[maybe_unused]] ComponentBase* child) {}
```

Comment:
This is absolutely the better way to solve the unused parameter error. However I decided against it, since it requires C++17. I'm not sure if FTXUI requires C++17 anyway, but if not this could be a breaking change for projects which need to "migrate" to C++17.

Reply:
FTXUI requires C++17 privately and publicly.. So we are fine.